Dockerfile

```diff
@@ -1,7 +1,11 @@
+# Use the distroless/cc:latest image as the base image
```

Comment:
Not sure it's a good idea to duplicate the image name in the comment, as they could get out of sync. Otherwise I like this PR!